examples/portfolio/src/types.ts

```diff
@@ -1,8 +1,8 @@
 export interface Project {
 	title: string;
-	client: string;
 	description: string;
 	publishDate: string;
 	tags: string[];
 	img: string;
+	img_alt?: string;
 }
```

Comment:
We’re already using this frontmatter interface, so maybe content collections would make sense.